data_management/mysqldumpall/README.md

```diff
@@ -12,7 +12,7 @@ A mysqldump that create database dumps in a certain way.
 ============================
 ## USAGE:
 ```bash
-bash dumpall.sh [-f] TARGET_DIRECTORY
+bash dumpall.sh -f TARGET_DIRECTORY
```

Comment:
Better to leave this un-changed. The [bracket] notation is used to indicate that this part is option, i.e., the user can use `-f` if they want, but it's not required, unlike `TARGET_DIRECTORY`, which is required.

Reply:
Done fixing